demo/wp-feature-api-demo/src/hooks/use-client-action-handler.ts

```diff
@@ -0,0 +1,167 @@
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import type { Message, ClientAction } from '../context/conversation-provider';
+
+/**
+ * Custom hook to handle client actions from the server
+ *
+ * @param addMessage   Function to add a message to the conversation
+ * @param setIsLoading Function to set loading state
+ * @return Object with functions to handle client actions
+ */
+export const useClientActionHandler = (
+	addMessage: ( message: Message ) => void,
+	setIsLoading: ( isLoading: boolean ) => void
+) => {
+	const executeClientFeature = useCallback(
+		async ( featureId: string, args: any ): Promise< unknown > => {
+			// Access store via global wp.data
+			// @todo We should probably export our feature store as a proper package and use that instead,
+			// this is to avoid double bundling by calling things from the root src/client/... files directly
+			const selector = ( window.wp as any )?.data?.select(
```

Comment:
We could avoid having the window.wp global usage. For example by importing the useRegistry:
import { useRegistry } from '@wordpress/data';
And then doing registry.select...

Reply:
This was a temporary workaround for some issues I was having accessing the store properly, but this gets fixed in my follow-up PR: https://github.com/Automattic/wp-feature-api/pull/19/files#diff-2ad67c14aab9bf8b2de50ebdd1ef4c360545c084739e2b0ed2405c7adb84d140